services/request-header.go

```diff
@@ -14,6 +14,8 @@ import (
 )
 
 // RequestHeader represents a Request Header in each services.
+// Part 4
```

Comment:
In this way, GoDoc won't be so good in appearance point of view.

![image](https://user-images.githubusercontent.com/6134010/44337826-f074f500-a4b6-11e8-9a40-1fdc0e5d39a9.png)

How do you think writing like this instead?

```go
// RequestHeader represents a Request Header in each services.
//
// Specicication: Part 4, 7.28
type RequestHeader struct {
```

![image](https://user-images.githubusercontent.com/6134010/44337875-1ef2d000-a4b7-11e8-9c87-96a6b60f7e80.png)